src/routes/transactions/helpers/twap-order.helper.ts

```diff
@@ -167,6 +177,20 @@ export class TwapOrderHelper {
     });
   }
 
+  /**
+   * Checks if the app associated contained in fullAppData is allowed.
+   *
+   * @param fullAppData - object to which we should verify the app data with
+   * @returns true if the app is allowed, false otherwise.
+   */
+  public isAppAllowed(fullAppData: FullAppData): boolean {
```

Comment:
This duplicates that in the `SwapOrderHelper`. Maybe we should extract it somehow - what do you think? (I created a `disallow-twap-mapping` branch before going away that might be possible to reference.)